### ISSUE TITLE
The empty val set and test set

### ISSUE THREAD

**Issue by JunXieFront** [sgrvinod/a-PyTorch-Tutorial-to-Transformers#4]
Thanks for your tutotial of _Attention is all you need_, and I have a small question.I would really appreciate for an answer.Because the project only has datasets of training, why we need a val_loader in _train.py_.Should I download data for val set by myself？
`    val_loader = SequenceLoader(
        data_folder=data_folder,
        source_suffix='en',
        target_suffix='de',
        split='val',
        tokens_in_batch=tokens_in_batch
    )`
